### PR TITLE
Lits documents

### PR DESCRIPTION
**Pull request by trangkarol**

### CODE REVIEW

**Comment by trangkarol** [comment]
screenshot from 2017-10-25 17-48-07

**Comment by trangkarol** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -13,10 +13,16 @@ def update
     end
   end
 
+  def document
+    # byebug
```
Comment (hhhhuy112) [inline app/controllers/users_controller.rb]: Xóa này đi chị ới

File: app/views/documents/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<div class="container-stories">
+  <div class="col-md-8">
+    <div class="page-header" id="stories">
+      <h1 class="text-primary text-center"><%= t "document.list_documents" %></h1>
+    </div>
+    <div class="col-md-6" style="margin-bottom: 10px;">
+      <div class="row">
+        <div class="col-md-6">
+        <%= image_tag "image3.jpg", class: "img-responsive"%>
```
Comment (hhhhuy112) [inline app/views/documents/show.html.erb]: space

File: app/views/users/document.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<%= render "header" %>
+<div class="container-stories">
+  <div class="container">
+    <%= render "left_bar" %>
+    <div class="col-md-9" id="div-show-user">
+      <% @list_document.each do |document| %>
+        <div class="col-md-12" style="margin-bottom: 10px;">
+          <div class="col-md-3">
+            <%= image_tag "image3.jpg", class: "img-responsive"%>
```
Comment (hhhhuy112) [inline app/views/users/document.html.erb]: space

**Comment by hhhhuy112** [comment]
commented

**Comment by trangkarol** [comment]
ready

**Comment by hhhhuy112** [comment]
merged

**Pull request merged**